Minor doc updates (#175)
jbms/sphinx-immaterial: docs/customization.rst

        - The :python:`"icon"` field can be specifed as `any of the icons bundled with this theme`_
        - The :python:`"link"` field is simply the hyperlink target added to the icon specified.
        

>         Optionally, custom text used for the icon's tooltip by specifying a :python:`"name"`

Reviewer: Optionally, custom text may be specified for the icon's tooltip with the :python:`"name"` field.

>         Optionally, custom text used for the icon's tooltip by specifying a :python:`"name"`

Author: ```suggestion
        Optionally, custom text may be specified for the icon's tooltip with the :python:`"name"`
```
